vultr.php

```diff
@@ -1396,6 +1405,18 @@ public function editService($package, $service, array $vars = null, $parent_pack
                     $this->log('api.vultr.com|backup_enable', serialize($params), 'input', true);
                     $this->parseResponse($vultr_api->backupEnable($params));
                 }
+            } elseif ($enable_backup
+                && $enable_backup->option_value == 'enable'
+                && (isset($vars['staff_id']) || $enable_backup->option_editable)
```

Comment:
I'm not sure this check is reliable. A client could just pass a 'staff_id' field in POST and be able to disable it even if they don't have access to do so via the option permission. 

Blesta already filters options settable by the client and passes the existing option as-is. It would probably be better to check that the config option was provided and differs from the existing value (i.e. went from enabled to disabled) and then perform this action.

Reply:
Also, even if the client did do so, it would only be to their disadvantage to lose backups.

Since it is a checkbox that is not reliable, since it will not be set when unchecked.

Reply:
It sounds like using that config option as a checkbox is not a good solution in this case. A Select option would be better because the option will not be removed from the service and then be available again to add like a checkbox would

Reply:
Alright, update to require the option be given to do a change.